src/nameutil.rs

```diff
@@ -149,6 +150,34 @@ pub fn shared_libs_to_links(shared_libs: &[String]) -> String {
     panic!("empty list of shared library");
 }
 
+pub fn use_glib_type(env: &crate::env::Env, import: &str) -> String {
+    format!(
+        "{}::{}",
+        if env.library.is_glib_crate() {
+            "crate"
+        } else {
+            "glib"
+        },
+        import
+    )
+}
+
+pub fn use_glib_if_needed(env: &crate::env::Env, import: &str) -> String {
+    format!(
+        "{}{}",
+        if env.library.is_glib_crate() {
+            ""
```

Comment:
Because in some cases (a lot of them actually), it's already imported in the current scope. Dark gir magic... Once we remove all imports, this function will very likely be useless.

Reply:
Why is that a problem to use `crate::` then?

Reply:
Honestly, no idea. I tried to keep the code as close to the original as possible. I think this is too much for just one PR. I can open an issue to investigate and remove it if needed (I think it's too much now but who knows?).

Reply:
Create an issue then, yes. That seems strange :)

Reply:
https://github.com/gtk-rs/gir/issues/999